api/src/main/java/org/hawkular/apm/api/model/trace/Trace.java

```diff
@@ -72,18 +75,42 @@ public Trace() {
     }
 
     /**
-     * @return the id
+     * @return the traceId
+     */
+    public String getTraceId() {
+        return traceId;
+    }
+
+    /**
+     * This method sets the trace id. When the fragment and trace id
```

Comment:
Aren't the end users developers? 

BTW. Do we publish this javadoc somewhere? If not probably users/devs are looking directly to source code and having nice documentation directly on fields can help, rarher than looking at various getters/setters. I raised this because when I started with APM I was having similar problem..